backend/src/test/java/com/hackner/musicompass/service/ArtistSearchServiceTest.java

```diff
@@ -0,0 +1,192 @@
+    private final ArtistSearchService artistSearchService =  new ArtistSearchService(discogsArtistApiService);
+
+    @Test
+    @DisplayName("Search for artist by search term returns an empty list")
+    public void searchForArtistBySearchTermEmptyList(){
+        //GIVEN
+        String searchTerm = "bliblablubb";
+
+        DiscogsArtistSearchResults discogsArtistSearchResults = DiscogsArtistSearchResults.builder()
+                .results(Collections.<DiscogsArtist>emptyList()).build();
+
+        when(discogsArtistApiService.getDiscogsArtistListBySearchTerm(searchTerm))
+                .thenReturn(discogsArtistSearchResults);
+
+        //WHEN
+        List<Artist> actual = artistSearchService.getArtistInfoListBySearchTerm(searchTerm);
+
+        //THEN
+        assertThat(actual.isEmpty(), is(true));
+    }
+
+    @Test
+    @DisplayName("Search for artist by search term returns a list with less than 5 entries")
+    public void searchForArtistBySearchTermShortList(){
+        //GIVEN
+        String searchTerm ="Hans";
+
+        DiscogsArtist discogsArtist1 = DiscogsArtist.builder()
+                .discogsArtistId("111")
+                .artistName("Hans Hammer")
+                .artistImageUrl("HammerImageUrl")
+                .discogsArtistUrl("HammerUrl").build();
+        DiscogsArtist discogsArtist2 = DiscogsArtist.builder()
+                .discogsArtistId("222")
+                .artistName("Hans Sichel")
+                .artistImageUrl("SichelImageUrl")
+                .discogsArtistUrl("SichelUrl").build();
+        DiscogsArtist discogsArtist3 = DiscogsArtist.builder()
+                .discogsArtistId("333")
+                .artistName("Hans Bohrer")
+                .artistImageUrl("BohrerImageUrl")
+                .discogsArtistUrl("BohrerUrl").build();
+
+        Artist artist1 = Artist.builder()
+                .artistName("Hans Hammer")
+                .artistInfo(new ArtistInfo().builder()
+                        .artistImageUrl("HammerImageUrl")
+                        .discogsArtistId("111")
+                        .discogsArtistUrl("HammerUrl")
+                        .build())
+                .build();
+        Artist artist2 = Artist.builder()
+                .artistName("Hans Sichel")
+                .artistInfo(new ArtistInfo().builder()
+                        .artistImageUrl("SichelImageUrl")
+                        .discogsArtistId("222")
+                        .discogsArtistUrl("SichelUrl")
+                        .build())
+                .build();
+        Artist artist3 = Artist.builder()
+                .artistName("Hans Bohrer")
+                .artistInfo(new ArtistInfo().builder()
+                        .artistImageUrl("BohrerImageUrl")
+                        .discogsArtistId("333")
+                        .discogsArtistUrl("BohrerUrl")
+                        .build())
+                .build();
```

Comment:
you could also move this to a helper method that returns a list of test artists and use it in both tests